To: vim_dev@googlegroups.com
Subject: Patch 7.3.035
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.035 (after 7.3.034)
Problem:    Stray semicolon after if statement. (Hari G)
Solution:   Remove the semicolon.
Files:	    src/os_win32.c


*** ../vim-7.3.034/src/os_win32.c	2010-10-23 14:02:48.000000000 +0200
--- src/os_win32.c	2010-10-24 14:31:04.000000000 +0200
***************
*** 231,237 ****
  	     * "!xxd" it's found in our starting directory.  Needed because
  	     * SearchPath() also looks there. */
  	    p = mch_getenv("PATH");
! 	    if (STRLEN(p) + STRLEN(exe_path) + 2 < MAXPATHL);
  	    {
  		STRCPY(temp, p);
  		STRCAT(temp, ";");
--- 231,237 ----
  	     * "!xxd" it's found in our starting directory.  Needed because
  	     * SearchPath() also looks there. */
  	    p = mch_getenv("PATH");
! 	    if (STRLEN(p) + STRLEN(exe_path) + 2 < MAXPATHL)
  	    {
  		STRCPY(temp, p);
  		STRCAT(temp, ";");
*** ../vim-7.3.034/src/version.c	2010-10-23 14:02:48.000000000 +0200
--- src/version.c	2010-10-24 14:32:04.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     35,
  /**/

-- 
    [clop clop]
ARTHUR:  Old woman!
DENNIS:  Man!
ARTHUR:  Man, sorry.  What knight lives in that castle over there?
DENNIS:  I'm thirty seven.
ARTHUR:  What?
DENNIS:  I'm thirty seven -- I'm not old!
                                  The Quest for the Holy Grail (Monty Python)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
